src/components/ttt-tile/ttt-tile.jsx

```diff
@@ -0,0 +1,59 @@
+var classNames = require('classnames');
+var React = require('react');
+var FormattedMessage = require('react-intl').FormattedMessage;
+
+require('./ttt-tile.scss');
+
+var TTTTile = React.createClass({
+    type: 'TTTTile',
+    propTypes: {
+        title: React.PropTypes.string.isRequired,
+        description: React.PropTypes.string.isRequired,
+        imageURL: React.PropTypes.string.isRequired,
+        tutorialLoc: React.PropTypes.string.isRequired
+    },
+    handleClick: function () {
+        alert('show Modal');
+    },
+    handleTutorialClick: function () {
+        alert('goto tutoriallink');
+    },
+    render: function () {
+        var classes = classNames(
+            'ttt-tile',
+            this.props.className
+        );
+        return (
+            <div className={classes} >
+                <a href={this.props.tutorialLoc}>
+                    <div className="ttt-tile-tutorial">
+                        <div className="ttt-tile-img-container">
```

Comment:
Clarifying - there are two places where I'm calculating the column/tile width. Would it be ok, to add `$tile-width: 18.75rem` to frameless.scss and use that in both places? Or is it better to use 18.75rem in the places where I'm setting the width along with the comment that $cols4 isn't working.